dist/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <title>Get news page</title>
+        <link rel="stylesheet" href="./css/style.css" />
+        <link rel="stylesheet" href="./css/normalize.css" />
+    </head>
+    <body>
+        <div class="wrapper-container">
+            <header>
+                <h1 class="news-header">News Page</h1>
+                <nav>
+                    <ul id="nav-list" class="nav-list">
+                        <li class="nav-item active" data-source="google-news">
+                            Google News
+                        </li>
+                        <li class="nav-item" data-source="bbc-news">
+                            BBC News
+                        </li>
+                        <li class="nav-item" data-source="cnn">CNN</li>
+                        <li class="nav-item" data-source="mtv-news">
+                            MTV News
+                        </li>
+                        <li class="nav-item" data-source="time">Time</li>
+                    </ul>
+                </nav>
+            </header>
+            <main id="main-container" class="main-container"></main>
+            <footer>
+                <div class="copyright">
+                    <a href="https://newsapi.org">&copy;Powered by News API</a>
+                </div>
+            </footer>
+        </div>
+        <script src="./js/polyfill/polyfill.min.js"></script>
```

Comment:
Imagine that you need to change source `index.html`. How will you move these changes to `dist` folder? Manually?
I don't see any build step for this.